win32_event_log/README.md

```diff
@@ -252,7 +272,39 @@ The legacy method is the default mode for filters.
 <!-- xxz tab xxx -->
 <!-- xxx tab "Logs" xxx -->
 
-For each filter, add a log processing rule in the configuration file at `win32_event_log.d/conf.yaml`.
+You can use the `query`, as well as the `log_processing_rules` regex option, to filter event logs. If you are using the Logs Tailer, the `query` option has better filtering performance than the `log_processing_rules` filters. The Datadog Agent cannot see any events filtered out by the `query` option. When using the `log_processing_rules` filters, the Agent can see every event and must process and format the event before applying the regex filter, so it takes more processing, and is less performant.
+
+You can use the `query` to filter events or filter out the bulk of the events with an [XPATH or structured XML query][21]. There is an expression limit on the syntax of XPath and XML queries. For additional filtering, use `log_processing_rules` filters.
```

Comment:
```suggestion
You can use the `query` to filter events or filter events with an [XPATH or structured XML query][21]. The `query` option can reduce the number of events that are processed by `log_processing_rules`, improving performance. There is an expression limit on the syntax of XPath and XML queries.  For additional filtering, use `log_processing_rules` filters.
```

Reply:
@clarkb7  and @estherk15 what is resulting text for the "query". Looks like the are a few versions. This is super important piece of information @clarkb7 and I would like to communicate efficiently and that implications and even reason bit more clear. That is why I am interested to read the whole section before commenting.